### ISSUE TITLE
High Contrast theme does not look good under Linux Gnome Dark Style.

### ISSUE THREAD

**Issue by HubKing** [dbeaver/dbeaver#21895]
### Description

The current High Contrast theme's colours are not good in my opinion. Probably a good idea to add a new "colours and fonts" preset for this theme.

1. UI elements' outlines/borders can be white. Currently, they are dark grey, almost invisible.
![image](https://github.com/dbeaver/dbeaver/assets/10900489/0377d7e7-b392-4795-bf4c-26f74b6a9e13)

2. Null is black.
![image](https://github.com/dbeaver/dbeaver/assets/10900489/ce6a8cd2-f304-4f1a-a171-a614d176621c)

3. The SQL query is not bright enough. Should be a lighter green colour.
![image](https://github.com/dbeaver/dbeaver/assets/10900489/5e746439-e1e7-435a-9f0b-87598abaf9ab)

4. The value should be white text on black background.
![image](https://github.com/dbeaver/dbeaver/assets/10900489/ad3e09e2-9dd7-48de-b14d-f0ef9c0b311e)

5. This thing is white text on white background.
![image](https://github.com/dbeaver/dbeaver/assets/10900489/6c4b562c-fc1b-4c11-9743-be057a9740ce)

6. In the column editor the `[ ]`/ `[v]` becomes black when selected.
![image](https://github.com/dbeaver/dbeaver/assets/10900489/cd82bfd6-757e-4df7-8279-6f68dc3e2808)

And so on... these are things I could find after using it around 10 minutes, and there should be more problems.

### DBeaver Version

23.2.2

### Operating System

Linux, Arch, Gnome, Wayland, Dark Style

### Database and driver

Sample SQLite DB.

### Steps to reproduce

1. Reset settings
2. Set "High Contrast" theme in settings.
3. Try to use the UI.

![image](https://github.com/dbeaver/dbeaver/assets/10900489/07691809-ad3d-4a4f-8a25-3671e066c9a2)

### Additional context

- **GNOME Version:**                               45.1
- **Windowing System:**                            Wayland

![image](https://github.com/dbeaver/dbeaver/assets/10900489/c5ff0555-2730-42f0-998e-c991da74b772)

![image](https://github.com/dbeaver/dbeaver/assets/10900489/308a67ad-3159-4f04-83fa-f4f463a12359)

**Comment by E1izabeth** [comment]
Thanks!
